<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="wsb-007-74" failures="1" tests="4" name="testInkAmountAfterTyping" time="0.002" errors="0" timestamp="2022-01-29T18:56:26 MSK">
  <testcase classname="CheckingInkAmountAfterTypingTest" name="testInkAmountAfterTyping" time="0.0"/>
  <testcase classname="CheckingInkAmountAfterTypingTest" name="testInkAmountAfterTyping" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [-1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [-1] but found [0]
at CheckingInkAmountAfterTypingTest.testInkAmountAfterTyping(CheckingInkAmountAfterTypingTest.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testInkAmountAfterTyping -->
  <testcase classname="CheckingInkAmountAfterTypingTest" name="testInkAmountAfterTyping" time="0.0"/>
  <testcase classname="CheckingInkAmountAfterTypingTest" name="testInkAmountAfterTyping" time="0.0"/>
</testsuite> <!-- testInkAmountAfterTyping -->
